docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -0,0 +1,971 @@
+
+#### How to get there
+
+In **Step 2** you have already adjusted the base Helm values file `camunda-values.yml` in `aws/dual-region/kubernetes` with the same changes as for the failover deployment for the environment variables.
+
+- `ZEEBE_BROKER_CLUSTER_INITIALCONTACTPOINTS`
+- `ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION0_ARGS_URL`
+- `ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION1_ARGS_URL`
+
+1. From the terminal context of `aws/dual-region/kubernetes`, you will do a Helm upgrade to update the existing Zeebe deployment in `CAMUNDA_NAMESPACE_0` to point to the failover Elasticsearch instance:
+
+```bash
+helm upgrade $HELM_RELEASE_NAME camunda/camunda-platform \
+  --version $HELM_CHART_VERSION \
+  --kube-context $CLUSTER_0 \
+  --namespace $CAMUNDA_NAMESPACE_0 \
+  -f camunda-values.yml \
+  -f region0/camunda-values.yml
+```
+
+#### Verification
+
+The following command will show the deployed pods of the healthy namespace. You should see that the Zeebe brokers have just restarted or are still restarting due to the configuration upgrade.
+
+```bash
+kubectl --context $CLUSTER_0 get pods -n $CAMUNDA_NAMESPACE_0
+```
+
+Alternatively, you can check that the Elasticsearch value was updated in the [StatefulSets](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) configuration of the Zeebe brokers and are reflecting the previous output of the script `generate_zeebe_helm_values.sh` in **Step 2**.
+
+```bash
+kubectl --context $CLUSTER_0 get statefulsets $HELM_RELEASE_NAME-zeebe -oyaml -n $CAMUNDA_NAMESPACE_0 | grep -A1 'ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION[0-1]_ARGS_URL'
+```
+
+<details>
+  <summary>Example Output</summary>
+  <summary>
+
+```bash
+  - name: ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION0_ARGS_URL
+    value: http://camunda-elasticsearch-master-hl.camunda-primary.svc.cluster.local:9200
+--
+  - name: ZEEBE_BROKER_EXPORTERS_ELASTICSEARCHREGION1_ARGS_URL
+    value: http://camunda-elasticsearch-master-hl.camunda-primary-failover.svc.cluster.local:9200
+```
+
+  </summary>
+</details>
+
+</div>
+  </TabItem>
+</Tabs>
+
+### Failback
+
+<Tabs queryString="failback">
+  <TabItem value="step1" label="Step 1" default>
+
+#### Deploy Camunda 8 in Failback Mode in Newly Created Region
+
+<StateContainer
+current={<Seven viewBox="140 40 680 500" />}
+desired={<Nine viewBox="140 40 680 500" />}
+/>
+
+<div>
+
+#### Current
+
+You have temporary Zeebe brokers deployed in failover mode together with a temporary Elasticsearch within the same surviving region.
+
+#### Desired
+
+You want to restore the dual-region functionality again and deploy Zeebe in failback mode to the newly restored region.
+
+Failback mode means that two brokers will be added to the cluster to allow processing and restore data. While two brokers are sleeping since you still have the temporary setup that you have to transfer.
+
+An Elasticsearch will also be deployed but not used yet since you have to restore a backup from the temporary setup.
```

Comment:
Just to clear up a misconception. The backup of the surviving ES cluster is used.
The failover one only contains a subset of data and is insufficient since if you lose the surviving region in the future, you'll be missing all the data from before the last incident as failover only contains from incident forward.